src/sentry/tsdb/snuba.py

```diff
@@ -85,6 +85,24 @@ class SnubaTSDB(BaseTSDB):
             "times_seen",
             [["outcome", "=", outcomes.Outcome.FILTERED]],
         ),
+        TSDBModel.organization_total_bytes_received: SnubaModelQuerySettings(
+            snuba.Dataset.Outcomes,
+            "org_id",
+            "bytes_received",
```

Comment:
I am not sure I get where we use sum instead of count as aggregation function for these cases when querying snuba.